tests/cython/bindlink/test_functions.py

```diff
@@ -61,3 +61,16 @@ def stop_go(atom):
         assert(false)
 
     return TruthValue(0,0)
+
+
+def func_one(v):
+    thing = ConceptNode("barleycorn")
+    thang = ListLink(thing, v)
+    thang.tv = TruthValue(0.9, 1)
+    return thang.tv
+
+
+func_two_result = TruthValue (0,1) # false
+def func_two(v):
+    return func_two_result
```

Comment:
This should be
```
def func_two(v):
    return ListLink(ConceptNode("barleycorn"), v).tv
```
which returns the TV that `func_one` had computed.  This proves that the atoms created by `func_one` are still accessible to `func_two`, i.e. it proves that func_one and func_two are sharing the same temporary atomspace.

Reply:
fixed. OK